extension_helpers/_setup_helpers.py

```diff
@@ -85,6 +111,24 @@ def get_extensions(srcdir='.'):
                         [os.path.join(main_package_dir, '_compiler.c')])
         ext_modules.append(ext)
 
+    use_limited_api = False
+    abi = get_distutils_option('py_limited_api', ['bdist_wheel'])
+    if abi:
+        version_info = _abi_to_version_info(abi)
+        if version_info:
```

Comment:
This isn't raising an error/warning if `py_limited_api` is specified but does not conform to the pattern, however, [`pypa/wheel` will raise an error in this case](https://github.com/pypa/wheel/blob/43fcdfda8a224918eb846f8aa4f2dbe0d440889d/src/wheel/bdist_wheel.py#L229-L232) so this should be fine.